.buildkite/pipeline.build_release.yml

```diff
@@ -37,18 +37,3 @@
     - python .buildkite/copy_files.py --destination branch_wheels --path ./.whl
     # Upload to latest directory.
     - if [ "$BUILDKITE_BRANCH" == "master" ]; then python .buildkite/copy_files.py --destination wheels --path ./.whl; fi
-
-- label: ":docker: Build Images: {{matrix}} - cpu"
```

Comment:
@can-anyscale This is required to complete this step in [the release process](https://www.notion.so/anyscale-hq/How-to-release-Ray-go-release-ray-28e4a2347577471288bbd7d64001fa88?pvs=4#9cc39bb28f5742778b62dd1381601e03). Do you know if is it just taken out temporally or if it's permanently?

Reply:
@GeneDer it is moved into .buildkite/build.rayci.yml (which is what civ2 used); just moving stuff around so this should have no affect on the release process :D; if it does anyway plz let me know, thankks

Reply:
I'm assuming you meant something like [this](https://buildkite.com/ray-project/premerge/builds/8041#018b19e7-42e1-400e-8999-05c8ab254207)? I think the new ones in civ2 works slightly differently and doesn't generate the dependency files we need to use to update the docs with. This is an example of [the old one](https://buildkite.com/ray-project/oss-ci-build-branch/builds/6386#018b07a3-bb76-42bc-bebe-55aaed0884f8), you can see in the artifacts tab it generates the files🙏

Reply:
Ah good catch @GeneDer , I'll put up a PR to fix this